src/main/pageobjects/page.ts

```diff
@@ -8,10 +8,7 @@ const baseURL = config.get('Environment.baseUrl');
 
 console.log('BASE URL : '+baseURL)
 export default class Page {
-    /**
-    * Opens a sub page of the page
-    * @param path path of the sub page (e.g. /path/to/page.html)
-    */
+    
```

Comment:
Why are these comments removed from Page class, please dont add anything in Page, unless it is needed